### ISSUE TITLE
Ctrl+A=>Left does not work

### ISSUE THREAD

**Issue by EggToaster** [corpnewt/ProperTree#182]
Web browsers lets you move to very left side of selection, so I can quickly comment to effectively disable stuff, but for some reason ProperTree does not have this and just moves cursor to right.

**Comment by corpnewt** [comment]
I'm not sure I understand what you are looking for. Are you able to provide more context or an example that displays the intended behavior?

-CorpNewt

**Comment by EggToaster** [comment]
Normal screen
![Screenshot_20240922-151829_crDroid Home.png](https://github.com/user-attachments/assets/7df2d5fa-dbe3-4c60-9605-f48f8fa04531)

Selecting something
![Screenshot_20240922-151834_crDroid Home.png](https://github.com/user-attachments/assets/91fe478c-be3e-47fc-8011-7f8e915f0d34)

Pressing left jumps to left edge of selection
![Screenshot_20240922-152312_Keep Notes.png](https://github.com/user-attachments/assets/f07aee9f-14ca-4167-a5c7-6c9c40bf350f)

Hi, this is an example of how this works on other applications.

**Comment by corpnewt** [comment]
Ahh, thanks for the clarification, that is on my to do list actually. I may be able to push something soonish for that.

-CorpNewt

**Comment by corpnewt** [comment]
The functionality should be present as of [this commit](https://github.com/corpnewt/ProperTree/commit/5ff659f5102af34ae024591d96b9e44c67b66100) - do note that I've only implemented it for the entries that pop up when you're editing keys/values in the current document.  At some point I may expand the text editing additions I've written to all the entries in the settings/conversion windows, but that's for a later date.

There appear to be some oddities when you don't bind `<Shift-Left>` and `<Shift-Right>` in addition to `<Left>` and `<Right>` respectively, so I had to include two placeholder methods which don't actually do anything in order to prevents some weird behaviors.

Hopefully that works as expected.

-CorpNewt
